source/code/plugins/filter_scom_cor_match.rb

```diff
@@ -1,5 +1,7 @@
+require_relative 'scom_common'
+
 module Fluent
-  class SCOMCorrelatedMatchFilter < Filter
+  class SCOMCorrelatedMatchFilter < SCOMTimerFilterPlugin
```

Comment:
It might be better if these plugin changes were separated out into a different PR from the scom service changes, but that's my opinion.